### PR TITLE
dialect/sql/builder: added FullJoin

### PR DESCRIPTION
**Pull request by Liooo**
please review 🙏

### CODE REVIEW

File: dialect/sql/builder.go [hunk 1]
```diff
@@ -2331,6 +2331,11 @@ func (s *Selector) RightJoin(t TableView) *Selector {
 	return s.join("RIGHT JOIN", t)
 }
 
+// FullJoin appends a `Full JOIN` clause to the statement.
```
Comment (a8m) [inline dialect/sql/builder.go:2334]:
```suggestion
// FullJoin appends a `FULL JOIN` clause to the statement.
```

**Review by a8m — APPROVED** [review]
💯 

Thanks for the contribution , @Liooo

File: dialect/sql/builder.go (hunk 1 above)
Comment (Liooo) [reply dialect/sql/builder.go]: 🙏

**Review by a8m — APPROVED** [review]
🚀

**Pull request merged**